tests/NCronJob.Tests/TestHelper.cs

```diff
@@ -21,6 +21,8 @@ private readonly CancellationTokenSource cancellationTokenSource
     protected ServiceCollection ServiceCollection { get; }
     protected FakeTimeProvider FakeTimer { get; } = new() { AutoAdvanceAmount = TimeSpan.FromMilliseconds(1) };
     protected Storage Storage { get; } = new();
+    protected IList<ExecutionProgress> Events { get; private set; } = [];
+    private readonly List<Action> disposers = [];
```

Comment:
Minor detail - but wouldn't it make it easier to capture the `IDisposable` itself and just call `Dispose` rather than the Action delegate?